### PR TITLE
Add Api Section

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**
Add API Page updated with pop-up

### CODE REVIEW

File: src/components/AddApiPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+
+import { useFormInputs, useHttpRequest } from "../hooks";
+import { Fallback } from "../components";
+
+const initialState = {email: "",password: ""};
+
+const AddApiPopup: React.FC = () => {
+  const { loading } = useHttpRequest();
+  const { bind } = useFormInputs(initialState);
```
Comment (pablo-clueless) [inline src/components/AddApiPopup.tsx]: You need to destructure inputs from here to be able to get the values out.

File: src/components/AddApiPopup.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import { makeStyles } from "@mui/styles";
+
+
+import { useFormInputs, useHttpRequest } from "../hooks";
+import { Fallback } from "../components";
+
+const initialState = {email: "",password: ""};
+
+const AddApiPopup: React.FC = () => {
+  const { loading } = useHttpRequest();
```
Comment (pablo-clueless) [inline src/components/AddApiPopup.tsx]: `sendRequest`?, `error`?, `clearError`?

File: src/components/AddApiPopup.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  const { bind } = useFormInputs(initialState);
+  const classes = useStyles();
+
+  const [category, setCategory] = React.useState('');
+  const [team, setTeam] = React.useState('');
+
+  const handleCategoryChange = (event: SelectChangeEvent) => {
```
Comment (pablo-clueless) [inline src/components/AddApiPopup.tsx]: Why not just abstract everything to one function since they're both select events??

**Review by pablo-clueless — COMMENTED** [review]
Check for comments

**Review by taiwoAkindele — APPROVED** [review]

**Review by taiwoAkindele — COMMENTED** [review]
Where is the instance for the onClick?

**Review by taiwoAkindele — COMMENTED** [review]
This is nice but the button is not working

File: src/components/AddApiPopup.tsx (hunk 1 above)
Comment (Vinyl-Davyl) [reply src/components/AddApiPopup.tsx]: Alright, Resolved

File: src/components/AddApiPopup.tsx (hunk 3 above)
Comment (Vinyl-Davyl) [reply src/components/AddApiPopup.tsx]: Alright Noted

**Comment by Vinyl-Davyl** [comment]
> This is nice but the button is not working

Yes this is just the UI for now. would need an endpoint later

File: src/components/AddApiPopup.tsx (hunk 2 above)
Comment (Vinyl-Davyl) [reply src/components/AddApiPopup.tsx]: What do you mean by this pls?
Its just the UI for now

**Review by pablo-clueless — APPROVED** [review]

**Comment by PhilemonBrain** [comment]
@Vinyl-Davyl please update this pr

**Comment by Vinyl-Davyl** [comment]
> @Vinyl-Davyl please update this pr

PR Updated already with latest changes

**Review by taiwoAkindele — APPROVED** [review]

**Pull request merged**